Ladybird/AndroidPlatform.cpp

```diff
@@ -53,31 +54,25 @@ void extract_ladybird_resources()
     if (file_or_error.is_error()) {
         qDebug() << "Unable to open test file file as expected, extracting asssets...";
 
-        MUST(extract_tar_archive(DeprecatedString::formatted("{}/ladybird-assets.tar", s_serenity_resource_root), s_serenity_resource_root));
+        MUST(extract_tar_archive(MUST(String::formatted("{}/ladybird-assets.tar", s_serenity_resource_root)), s_serenity_resource_root));
     } else {
         qDebug() << "Opened app-browser.png test file, good to go!";
         qDebug() << "Hopefully no developer changed the asset files and expected them to be re-extracted!";
     }
 }
 
-ErrorOr<void> extract_tar_archive(DeprecatedString archive_file, DeprecatedString output_directory)
+ErrorOr<void> extract_tar_archive(String archive_file, DeprecatedString output_directory)
 {
     constexpr size_t buffer_size = 4096;
 
-    auto file = TRY(Core::DeprecatedFile::open(archive_file, Core::OpenMode::ReadOnly));
+    auto file = TRY(Core::InputBufferedFile::create(TRY(Core::File::open(move(archive_file), Core::File::OpenMode::Read))));
 
     DeprecatedString old_pwd = TRY(Core::System::getcwd());
 
-    TRY(Core::System::chdir(output_directory));
+    TRY(Core::System::chdir(move(output_directory)));
```

Comment:
This takes a stringview, why are you moving?

Reply:
ah, blind move-ing because I saw we didn't need the variable anymore, and clang-tidy complained I was "copying" it. I'll take a look at the other move and remove if needed too.